collector/container/Dockerfile

```diff
@@ -9,6 +9,11 @@ ENV ROOT_DIR=$ROOT_DIR
 ENV COLLECTOR_VERSION="${COLLECTOR_VERSION}"
 ENV MODULE_VERSION="${MODULE_VERSION}"
 ENV COLLECTOR_HOST_ROOT=/host
```

Comment:
These changes are strictly needed because there existing functionality when COLLECTOR_PROFILING is enabled (https://github.com/stackrox/collector/blob/master/collector/lib/Profiler.h)

Reply:
You can also set the environment variables directly in the golang test, for example, `s.collector.Env["CPUPROFILE"]="/tmp/collector.prof"`

Reply:
I am now setting this in collector_manager.go.